lib/beaker/host/unix/pkg.rb

```diff
@@ -17,28 +17,31 @@ def check_for_command(name)
     end
   end
 
-  def check_for_package(name)
+  def check_for_package(name, opts = {})
+    opts = {:accept_all_exit_codes => true}.merge(opts)
+    opts[:prepend_cmds] ? pc = opts.select {|k,v| k == :prepend_cmds} :
```

Comment:
I believe that if we switch from `exec` to `execute` we can just pass the prepend_cmd through without having to do this initial processing.